frontend/src/components/graph/ArtifactNode.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FolderIcon from '@material-ui/icons/Folder';
+import React from 'react';
+import { Handle, Position } from 'react-flow-renderer';
+
+interface ArtifactNodeProps {
+  id: string;
+  data: any;
```

Comment:
nit: type data?

Reply:
Sounds good, will update this after https://github.com/kubeflow/pipelines/pull/6420 is merged.

Reply:
Updated